Apps/Src/Amps.c

```diff
@@ -84,11 +84,11 @@ ErrorStatus Amps_UpdateMeasurements(void) {
 
 /** Amps_CheckStatus
  * Checks if pack does not have a short circuit
- * @param chargingOnly a flag that indicates whether we should allow
- * 		  only charging or both charging and discharging.
+ * @param maxTemperature the maximum temperature measured in the most recent batch of temperature readings
  * @return SAFE or DANGER
  */
-SafetyStatus Amps_CheckStatus(bool chargingOnly) {
+SafetyStatus Amps_CheckStatus(int32_t maxTemperature) {
+	bool chargingOnly = maxTemperature > MAX_CHARGE_TEMPERATURE_LIMIT;
```

Comment:
Why is it just MAX_CHARGE_TEMPERATURE_LIMIT? Wouldn't we have to check the sign of the current to see whether it is the charge or discharge temperature limit?